Footer UI Revamp

yenaing-oo opened the pull request:
Closes https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/issues/509

yenaing-oo commented:
@danloveg 
If you notice repeated HTML, such as the social links and the UNESCO logo, that is intentional so that they can be hidden/displayed in different columns depending on the screen size.

danloveg commented:
> @danloveg If you notice repeated HTML, such as the social links and the UNESCO logo, that is intentional so that they can be hidden/displayed in different columns depending on the screen size.

Do you mind if I make a commit on this branch? I re-worked some of the CSS to make it work without requiring that extra UNESCO image

yenaing-oo commented:
> > @danloveg If you notice repeated HTML, such as the social links and the UNESCO logo, that is intentional so that they can be hidden/displayed in different columns depending on the screen size.
> 
> Do you mind if I make a commit on this branch? I re-worked some of the CSS to make it work without requiring that extra UNESCO image

I don't mind at all. Please go ahead.

danloveg commented:
> > > @danloveg If you notice repeated HTML, such as the social links and the UNESCO logo, that is intentional so that they can be hidden/displayed in different columns depending on the screen size.
> > 
> > 
> > Do you mind if I make a commit on this branch? I re-worked some of the CSS to make it work without requiring that extra UNESCO image
> 
> I don't mind at all. Please go ahead.

My new commit shows how you can make it look similar without hiding/showing that extra image. The only issue I left in there is that the images aren't arranged nicely when the screen is medium size but I think that could be addressed pretty easily

yenaing-oo commented:
> > > > @danloveg If you notice repeated HTML, such as the social links and the UNESCO logo, that is intentional so that they can be hidden/displayed in different columns depending on the screen size.
> > > 
> > > 
> > > Do you mind if I make a commit on this branch? I re-worked some of the CSS to make it work without requiring that extra UNESCO image
> > 
> > 
> > I don't mind at all. Please go ahead.
> 
> My new commit shows how you can make it look similar without hiding/showing that extra image. The only issue I left in there is that the images aren't arranged nicely when the screen is medium size but I think that could be addressed pretty easily

I see what you did there - you used a flexbox row in place of displaying the repeated image in a different column. I tried doing that initially but was only able to achieve placing both logos in a row on the left half of the screen.

I fixed a few things - I added back some styling because the logo placements were broken for the large screen as well. The placement of the logos on the medium and small screens should look right now, after overriding gap size and aligning to the left.

Let me know if you spot any more issues.

danloveg commented on app/recordtransfer/static/recordtransfer/css/base/footer.css:
```diff
@@ -1,108 +1,230 @@
+
+    .footer-column {
+        width: 100%;
+    }
+
+    .nctr-logo-container,
```
The logos in the footer aren't wrapping when they overwhelm the size of the column (pertains to the *small* screen size). Here's a suggestion to make them break nicely when the screen gets small enough that the images reach their `min-width` of 200px.

```suggestion
    }

    .footer-logo {
        gap: 20px;
        flex-wrap: wrap;
        align-items: center;
        justify-content: center;
    }

    .nctr-logo-container,
```

yenaing-oo replied:
That's a good catch - I've applied your suggestion with the only difference being that I used `row-gap` (for when items are in a column) instead of `gap` to maximize the use of space when the flexbox is a row layout.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/525/commits/ea0f738decb30a32abcb5b4e8e2133f978bcc3d7

danloveg commented on app/recordtransfer/templates/recordtransfer/footer.html:
```diff
@@ -1,111 +1,64 @@
+
+            <div class="contact-content">
+                <h3>Contact Us</h3>
+                <p>Chancellor's Hall, 177 Dysart Road,<br>
+                University of Manitoba, Winnipeg</p>
```
How do you feel about making this a link?

https://maps.app.goo.gl/S7cEoxgfSkxKrv836

yenaing-oo replied:
That's a good idea, I added it here. 

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/525/commits/9d97cd252d9c884d890f797eba010ff6bee955fd

I took some time to make the text aligned (particularly the address) after adding a location pin icon beside it.
